### PR TITLE
Add table of buildah commands to readme.md

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
This would help satisfy https://github.com/projectatomic/buildah/issues/13

**Comment by rhatdan** [comment]
@nalind PTAL

**Comment by rhatdan** [comment]
bot retest

**Comment by nalind** [comment]
We're having some network trouble today.

**Comment by nalind** [comment]
@rh-atomic-bot retry

File: README.md [hunk 1]
```diff
@@ -53,7 +53,23 @@ Then to install buildah follow the steps in this example:
   ./buildah --help
 ```
 
+## commands
```
Comment (TomSweeneyRedHat) [inline README.md]: Should 'commands' be capitalized?

**Comment by rhatdan** [comment]
@rh-atomic-bot r+	420cb15

**Pull request closed without merging**